modules/cnf-performing-end-to-end-tests-running-cyclictest.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-performing-platform-verification-latency-tests.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-end-to-end-tests-running-cyclictest_{context}"]
+= Running cyclictest
+
+The `cyclictest` tool measures the real-time kernel scheduler latency on the specified CPUs.
+
+[IMPORTANT]
+====
+**Always** run the latency tests with `DISCOVERY_MODE=true` set. If you don't, the test suite will make changes to the running cluster configuration.
+====
+
+[NOTE]
+====
+When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. To make the `podman` command work, append `:Z` to the volumes creation; for example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
+====
+
+.Prerequisites
+
+* You have logged in to `registry.redhat.io` with your Customer Portal credentials.
+
+* You have installed the real-time kernel in the cluster.
+
+* You have applied a cluster performance profile by using Node Tuning Operator.
```

Comment:
in 4.10 and below, the performance profile is applied using the performance addon operator, so PAO should be installed first and then apply a PP.